legacy/src/main/java/org/opensearch/sql/legacy/esdomain/LocalClusterState.java

```diff
@@ -6,7 +6,6 @@
 
 package org.opensearch.sql.legacy.esdomain;
 
-import static org.opensearch.common.settings.Settings.EMPTY;
 
```

Comment:
```suggestion
```

Reply:
Addressed in 65df31b37e9b2766d0a5fd4c31745fdc3638d4af